calico/felix/felix.py

```diff
@@ -192,6 +192,7 @@ def dump_top_level_actors(log):
 
 
 def main():
+    _log.debug("Felix main started.")
```

Comment:
The log statement comes before we've initialised the log (which is on the line below) so it'll likely just cause an error to stderr saying "no loggers found"